cocotb/triggers.py

```diff
@@ -623,24 +608,20 @@ def __repr__(self):
         return "{}({!r})".format(type(self).__name__, self._coroutine)
 
 
-class Waitable(object):
+class Waitable(Awaitable):
```

Comment:
Just a note - this doesn't actually have any effect other than making `issubclass(Waitable, Awaitable)` faster. It's good documentation though, so I'm not opposed.